.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
     "extends": "standard",
-    "installedESLint": true,
```

Comment:
?

Reply:
Removed it as it's a bug that's introduced on ESLint side. This property doesn't do anything useful.

Link to issue: https://github.com/eslint/eslint/issues/7758